terraform/modules/hub/hub_metadata.tf

```diff
@@ -41,6 +42,14 @@ data "template_file" "metadata_task_def" {
   }
 }
 
+module "metadata_ecs_roles" {
+  source = "modules/ecs_iam_role_pair"
+
+  deployment       = "${var.deployment}"
+  service_name     = "metadata"
+  tools_account_id = "${var.tools_account_id}"
```

Comment:
Is this for Concourse to be able to deploy?

Reply:
It is so we can get images from the tools ECR